packages/fast-development-site-react/app/app.tsx

```diff
@@ -123,10 +128,19 @@ export default class App extends React.Component<{}, AppState> {
             component: Paragraph,
             status: Status.alpha,
         };
+        const componentObj2: any[] = [{ text: "uno" }, { text: "due" }, { text: "tre" }];
```

Comment:
Can we rename these so that we have more of an indication on what they are, so this could be `spanTestComponents`, and can you change the ones above to `paragraphTestComponents`